### PR TITLE
Implementpeternovingalgorithm

### PR DESCRIPTION
**Pull request by jaimeHBarriga**

### CODE REVIEW

**Comment by jaimeHBarriga** [comment]
Carlos please proceed with the internal CR.

File: src/algorithms/peter_norving.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        return [key + value for key in key_list for value in value_list]
+
+    def parse_grid(self, grid):
+        """
+        Convert grid to a dict of possible values, {square: digits}, or
+        return False if a contradiction is detected.
+
+        Keyword arguments:
+        param grid -- Dict with initial values.
+        return a dictionarie with all the possible values per square.
```
Comment (ccatari) [inline src/algorithms/peter_norving.py:36]: there is a typo in the documentation 'dictionarie'.

**Comment by ccatari** [comment]
Looks good only one observation found.

**Comment by jaimeHBarriga** [comment]
I am going to fix the observations.

**Pull request closed without merging**